configs/components/rubygem-ffi.rb

```diff
@@ -1,6 +1,6 @@
 component "rubygem-ffi" do |pkg, settings, platform|
-  pkg.version '1.9.25'
-  pkg.md5sum "e8923807b970643d9e356a65038769ac"
+  pkg.version '1.12.2'
+  pkg.md5sum "28dc3d1294a04b728d24ba025e331b13"
```

Comment:
I know we've had some challenges with "ffi" versions in the past in PDK, I'll see if I can make an ad-hoc build with this PR and validate this works.

Reply:
Same with bolt on OSX. might be worth seeing that as well with this ffi

Reply:
@lucywyman we need to update the md5 sums for the windows version of the `ffi` gem a little further down in the file.

Reply:
@scotje Apologies, I dropped your commit since newer ffi breaks things on Windows (see comments below), so there's no need to update the checksums.

Reply:
No worries!